src/tracks/SampleTrack.cpp

```diff
@@ -495,9 +495,14 @@ void SampleTCOView::paintEvent( QPaintEvent * pe )
 	QColor c;
 	bool muted = m_tco->getTrack()->isMuted() || m_tco->isMuted();
 
-	// state: selected, muted, normal
-	c = isSelected() ? selectedColor() : ( muted ? mutedBackgroundColor() 
-		: painter.background().color() );
+	if (isSelected ())
+		c = selectedColor ();
+	else if (muted)
+		c = mutedBackgroundColor ();
+	else if (m_tco->isRecord ())
+		c = recordingBackgroundColor ();
+	else
+		c = painter.background ().color ();
```

Comment:
FYI, You don't need to add spaces between function names and parentheses.

Reply:
It's my IDE that do it automatically. I'll update it.